stacks/triliovault-operator/deploy.sh

```diff
@@ -112,11 +115,11 @@ access_tvk_ui () {
 install_license () {
   #This module is use to install license
   echo "Installing Freetrial license..."
-  kubectl apply -f tvk_install_license.yaml --namespace "$NAMESPACE"
+  kubectl apply -f "$TVK_PATH/tvk_install_license.yaml" --namespace "$NAMESPACE"
```

Comment:
`"$TVK_PATH/tvk_install_license.yaml"` will fail in the production environment. Please set the license path in a variable.

for eg. -
```
if [ -z "${MP_KUBERNETES}" ]; then
  # use local version of values.yml
  values="$TVK_PATH/values.yml"
  license="$TVK_PATH/tvk_install_license.yaml"
else
  # use github hosted master version of values.yml
  values="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/triliovault-operator/values.yml"
  license="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/triliovault-operator/tvk_install_license.yml
```

Reply:
Made the changes now to assigned
`TVK_LICENSE_FILE="$TVK_PATH/tvk_install_license.yaml"`
the new variable is used in kubectl command:
`kubectl apply -f "$TVK_LICENSE_FILE" --namespace "$NAMESPACE"`

Let me know if this should work since `$TVK_PATH` is deduced from `$ROOT_DIR`

Reply:
@keladhruv Can you please check if the above fix work in production env?